common/utils/src/main/resources/error/error-conditions.json

```
@@ -4186,6 +4186,12 @@
     ],
     "sqlState" : "42846"
   },
+  "UNKNOWN_PRIMITIVE_TYPE_IN_VARIANT" : {
+    "message" : [
+      "Unknown primitive type with id <id> was found in a variant value. The type might be supported in a newer version."
```

I think we can remove the second part.
```suggestion
      "Unknown primitive type with id <id> was found in a variant value."
```